dapr/proto/scheduler/v1/scheduler.proto

```diff
@@ -17,45 +15,114 @@ service Scheduler {
 	rpc DeleteJob(DeleteJobRequest) returns (DeleteJobResponse) {}
 	// WatchJobs is used by the daprd sidecar to connect to the Scheduler
 	// service to watch for jobs triggering back.
-	rpc WatchJobs(WatchJobsRequest) returns (stream WatchJobsResponse) {}
+	rpc WatchJobs(stream WatchJobsRequest) returns (stream WatchJobsResponse) {}
 }
 
-// TODO(Cassie): add an internal Job message to have a diff one than dapr.proto
+message Job {
+  // The schedule for the job.
+  optional string schedule = 1;
 
-message WatchJobsRequest {
-	// The app_id of the sidecar.
+  // Optional: jobs with fixed repeat counts (accounting for Actor Reminders).
+  optional uint32 repeats = 2;
+
+  // Optional: sets time at which or time interval before the callback is invoked for the first time.
+  optional string due_time = 3;
+
+  // Optional: Time To Live to allow for auto deletes (accounting for Actor Reminders).
+  optional string ttl = 4;
+
+  // Job data.
+  google.protobuf.Any data = 5;
+}
+
+// ScheduleJobMetadataApp is the message used by the daprd sidecar to schedule
+// a job from an App.
+message ScheduleJobMetadataSourceApp {}
```

Comment:
Can we make it something like: `ScheduleJob<type>`. its super long as it exists now. Then its more extendible for diff types of jobs if they need diff logical things done vs source. There are a limited number of 'sources' vs the potential expansion upon 'types'
```suggestion
message ScheduleTypeJob {}
```